addOns/grpc/src/main/java/org/zaproxy/addon/grpc/internal/ExtensionGrpcWebSocket.java

```diff
@@ -0,0 +1,87 @@
+package org.zaproxy.addon.grpc.internal;
```

Comment:
Same here.

Reply:
are you suggesting to change it to org.zaproxy.addon.grpc.internal.websocket, but then it is giving me error because there is not dir name websocket in internal dir

Reply:
Yes, you need to move it accordingly.